app/src/components/SearchInput/SearchInput.js

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+import PropTypes from 'prop-types';
+import './SearchInput.scss';
+
+const SearchInput = (props) => {
+  const { defaultValue, onSearch } = props;
+
+  const [searchValue, setSearchValue] = useState(defaultValue || '');
```

Comment:
Since in `MovieListPage.jsx` we  have state which controls search value it becomes redundant here. Try to rebuild component to be reactive without internal state.